cmd/create.go

```diff
@@ -31,7 +31,7 @@ including connectors.`,
 
 	createCmd.AddCommand((&CreateConnector{}).command())
 	createCmd.AddCommand(CreatePipelineCmd())
```

Comment:
Unrelated to this change, but one question I had: would it make sense to also expose logs from the `CreatePipelineCmd`? I'm not really sure if this would already automatically by applying the change from line 34 here as well or if this would require more changes - what do you think?

Reply:
@jayjayjpg great point. I could simply ship that change (one line), or refactor the command to include more test coverage like this one. I'll do the latter since I think it's worth doing. I'll create a jira ticket and will get it done now. Thanks again for the call out.

Reply:
@jayjayjpg I actually noticed we do have logs in that command https://github.com/meroxa/cli/blob/master/cmd/create_pipeline.go#L67 and https://github.com/meroxa/cli/blob/master/cmd/create_pipeline.go#L78. Did you maybe mean to something else?

Reply:
Ah sweet, yes that's exactly what I was looking for - thank you for outlining that!